.gitignore

```diff
@@ -0,0 +1,5 @@
+.score-compose/99-default.provisioners.yaml
```

Comment:
much better to ignore the entire directory right?

```
.score-compose/
```

Reply:
Good catch